cmd/compute-domain-kubelet-plugin/checkpoint.go

```diff
@@ -12,14 +12,16 @@ type Checkpoint struct {
 }
 
 type CheckpointV1 struct {
-	PreparedClaims PreparedClaims `json:"preparedClaims,omitempty"`
+	PreparedDevicesByClaimUID PreparedDevicesByClaimUID `json:"PreparedDevicesByClaimUID,omitempty"`
+	PreparedClaims            PreparedClaims            `json:"preparedClaims,omitempty"`
```

Comment:
Note to reviewers: watch out, I propose to use the old term `PreparedClaims`  for something new (it is a great name for what I want to introduce here: a collection of `resourceapi.ResourceClaim` objects).

Reply:
OK. This explains why the GPU plugin uses the NEW name -- since it doesn't have global claims like the compute domain driver.